engineering/teams/methodology-ml.md

```diff
@@ -0,0 +1,31 @@
+Machine Learning (ML) team follows the [common methodology](https://github.com/src-d/guide/blob/master/engineering/methodology.md) shared by
+the rest of the teams, with the following additions:
+
+- We meet two times per week, on Tuesday and on Thursday, to sync and discuss the status.
+- We meet once per month to do middle-term planning of the [OKRs](https://github.com/src-d/okrs) fulfillment. We fuse that meeting with a retrospective.
+- We minute every meeting, whether it is with people from academia or internal. The up-to-date order of scribes is in [ml-backlog](https://github.com/src-d/ml-backlog#minutes-scribe-order), they are chosen round robin.
+- We post our [OSD](../../talent/open_source_days.md) plans to 🔑`#osd` channel on Slack in the morning.
+- We mention the URLs of the issues/PRs we work on in our daily Geekbot standups.
+- We passionately, thoughtfully follow the [Python style guide](../conventions/python.md).
+- We do **not** use Anaconda.
+- All the team is running on Ubuntu. You can use other OS if you like adventures and want to spend time on solving unavoidable compatibility problems.
```

Comment:
I run on MacOS and Ubuntu both and everything is perfect :)

Reply:
This was not the case with @irinakhismatullina, I still remember those weird problems with Babelfish she had.